lib/virt_autotest/utils.pm

```diff
@@ -430,7 +467,7 @@ sub ensure_online {
             }
             # Check also if name resolution works - restart libvirtd if not
             if (script_run("ssh $guest ping -c 3 -w 120 $dns_host", timeout => 180) != 0) {
-                restart_libvirtd if (is_xen_host || is_kvm_host);
+                restart_libvirtd if (is_monolithic_libvirtd and (is_xen_host || is_kvm_host));
```

Comment:
For nearly all below libvirtd related operations restart/stop, the is_monolithic_libvirtd condition is added. Why? Modular libvirtd needs similar operations too. Do I miss any of your plans, eg implementing them in another PR?

Reply:
I agree with creating a new ticket to add a new function instead of extend the current restart_libvirtd(),